lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,41 @@
-module Minesweeper_2pl
+module Minesweeper
   class App
-    def call
-      "Hello World"
+    BOMB_PERCENT = 0.75
+    MAX_ROW_NUM = 20
+
+    attr_accessor :game, :cli
+
+    def initialize
+      cli = CLI.new
+      self.cli = cli
+      game_config = cli.start
+      game = Game.new(game_config.first, game_config.last)
```

Comment:
One great benefit of object oriented programming is that we can name almost everything. What are `game_config.first` and `game_config.last` here?

You could name these by unpacking `game_config` into two separate variables:

```rb
number_of_players, number_of_bombs = cli.start
```

Or even better, you could create a simple class/struct to encapsulate this information:

```rb
class GameConfig:
  attr_reader :players, :bombs

  def initialize(players, bombs)
    @players = players
    @bombs = bombs
  end
end
```